docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -33,6 +33,12 @@ If setting up an Airbyte server does not fit your use case needs (i.e. you're us
 
 ## 1: Install `abctl`
 
+:::tip
```

Comment:
Can you move this to be at the end of "Install Airbyte" instead of at the top of the section? We should recommend troubleshooting tips once they've tried it.